docs/interfaces/lite/readme.md

```diff
@@ -33,7 +33,7 @@ combination of module APIs, depending on which modules the state machine uses. T
 initially support [ICS0](https://cosmos.network/rpc/#/ICS0) (TendermintAPI), [ICS1](https://cosmos.network/rpc/#/ICS1) (KeyAPI), [ICS20](https://cosmos.network/rpc/#/ICS20) (TokenAPI), [ICS21](https://cosmos.network/rpc/#/ICS21) (StakingAPI),
 [ICS22](https://cosmos.network/rpc/#/ICS22) (GovernanceAPI) and [ICS23](https://cosmos.network/rpc/#/ICS23) (SlashingAPI).
 
-![high-level](./pics/high-level.png)
+<!-- ![high-level](./pics/high-level.png) -->
```

Comment:
all these images should be deleted if they are missing

Reply:
I mean, I left these as placeholders, because they might be helpful to add these images later down the road.

Reply:
let's delete them. We can add them again if we ever need to